cymetric/convenient_interface.py

```diff
@@ -0,0 +1,584 @@
+
+    if len(nuc_list) != 0:
+        nuc_list = format_nuclist(nuc_list)
+
+    df = get_inventory_df(evaler_, fac_list, nuc_list)
+
+    group_end = ['Time']
+    group_start = group_end + ['Quantity']
+    df = df[group_start].groupby(group_end).sum()
+    df.reset_index(inplace=True)
+
+    time = evaler_.eval('TimeList')
+    df = add_missing_time_step(df, time)
+    return df
+
+
+def get_inventory_activity_df(evaler_, fac_list=[], nuc_list=[]):
+    """
+    Get a simple time series of the activity of the inventory in the selcted
+    facilities. Applying nuclides selection when required.
+
+    Parameters
+    ----------
+    evaler_ : evaler
+    fac_name : name of the facility
+    nuc_list : list of nuclide to select.
+    """
+
+    if len(nuc_list) != 0:
+        nuc_list = format_nuclist(nuc_list)
+
+    df = get_inventory_df(evaler_, fac_list, nuc_list)
+    for i, row in df.iterrows():
+        val = 1000 * data.N_A * df.ix[i, 'Quantity'] * \
+            data.decay_const(int(df.ix[i, 'NucId']))
+        df.set_value(i, 'Activity', val)
+
+    return df
+
+
+def get_inventory_activity_timeseries(evaler_, fac_list=[], nuc_list=[]):
+    """
+    Get a simple time series of the decay heat of the inventory in the selcted
+    facilities. Applying nuclides selection when required.
+
+    Parameters
+    ----------
+    evaler_ : evaler
+    fac_name : name of the facility
+    nuc_list : list of nuclide to select.
+    """
+
+    if len(nuc_list) != 0:
+        nuc_list = format_nuclist(nuc_list)
+
+    activity = get_inventory_activity_df(evaler_, fac_list, nuc_list)
+    group_end = ['Time']
+    group_start = group_end + ['Activity']
+    activity = activity[group_start].groupby(group_end).sum()
+    activity.reset_index(inplace=True)
+
+    time = evaler_.eval('TimeList')
+    df = add_missing_time_step(df, time)
+    return activity
+
+
+def get_inventory_decayheat_df(evaler_, fac_list=[], nuc_list=[]):
+    """
+    Get a Inventory PDF including the decay heat of the inventory in the selected
+    facilities. Applying nuclides selection when required.
+
+    Parameters
+    ----------
+    evaler_ : evaler
+    fac_name : name of the facility
+    nuc_list : list of nuclide to select.
+    """
+
+    if len(nuc_list) != 0:
+        nuc_list = format_nuclist(nuc_list)
+
+    df = get_inventory_activity_df(evaler_, fac_list, nuc_list)
+    for i, row in df.iterrows():
+        val = data.MeV_per_MJ * \
+            df.ix[i, 'Activity'] * data.q_val(int(df.ix[i, 'NucId']))
+        df.set_value(i, 'DecayHeat', val)
+
+    return df
+
+
+def get_inventory_decayheat_timeseries(evaler_, fac_list=[], nuc_list=[]):
+    """
+    Get a simple time series of the decay heat of the inventory in the selcted
+    facilities. Applying nuclides selection when required.
+
+    Parameters
+    ----------
+    evaler_ : evaler
+    fac_name : name of the facility
+    nuc_list : list of nuclide to select.
+    """
+
+    if len(nuc_list) != 0:
```

Comment:
My understanding is that nuc_list is a filtering mechanism....but there is a warning in the ger_reduced_df function about having any of the filtering mechanisms empty: https://github.com/bam241/cymetric/pull/3/files#diff-908097c62f1520cddbe1acee59c44083R74

I think instead of that there should just be no filtering and all nucs/facilities/etc should be returned?